macros/get_problem_id.sql

```diff
@@ -3,6 +3,6 @@
 -- or is followed by '/answer' or '/hint'
 {% macro get_problem_id(object_id) %}
     regexpExtract(
-        object_id, 'xblock/([\w\d-\+:@]*@problem\+block@[\w\d][^_]*)(_\d_\d)?', 1
+        object_id, 'xblock/([\w\d-\+:@]*@problem\+block@[\w\d][^_\/]*)(_\d_\d)?', 1
```

Comment:
This regexp was not removing '/answer' from the end of the string which caused data to be excluded in subsequent queries